app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Registered Users</h1>
+
+  <% @users.each do |user| %>
+    <% if user.role == "member" %>
```

Comment:
You could also utilize the method you wrote on the User model here: `if !user.admin?` - your choice! Whatever you think is more readable.